backend/small_eod/institution/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+# Register your models here.
+
+from institution.models import Institution, AdministrativeUnit, ExternalIdentifier, AddressData
+
+admin.site.register(Institution)
```

Comment:
Warto to wprowadzić dla wszystkich modeli. Przyjmuje, że taka postać jest dobra na początek, chociaż samodzielne zarządzanie np. ```AddressData``` budzi moje wątpliwości (w mojej ocenie to powinno się dziać poprzez ```Institution```).

Reply:
Zrobione